dot/core/service_integration_test.go

```diff
@@ -82,9 +82,10 @@ func newTestDigest(t *testing.T, slotNumber uint64) scale.VaryingDataTypeSlice {
 	return vdts
 }
 
-func generateTestValidRemarkTxns(t *testing.T, pubKey []byte, accInfo types.AccountInfo) ([]byte, runtime.Instance) {
+func generateTestValidRemarkTxns(t *testing.T, genesisFilePath string,
+	pubKey []byte, accInfo types.AccountInfo) ([]byte, runtime.Instance) {
 	t.Helper()
-	projectRootPath := filepath.Join(utils.GetProjectRootPathTest(t), "chain/gssmr/genesis.json")
+	projectRootPath := filepath.Join(utils.GetProjectRootPathTest(t), genesisFilePath)
```

Comment:
Maybe add a function in `lib/utils` with a switch statement returning one of `GetGssmrGenesisRawPathTest`, `GetGssmrGenesisPathTest`, etc. depending on a string. Also ideally define each constant string as a custom type for code completion

```go
type GenesisName string

const (
    GssmrGenesis GenesisName = "gssmr genesis"
    GssmrGenesisRaw GenesisName = "gssmr genesis raw"
)
```

so you would have something like

```go
func generateTestValidRemarkTxns(t *testing.T, genesisName utils.GenesisName,
	pubKey []byte, accInfo types.AccountInfo) ([]byte, runtime.Instance) {
	t.Helper()
	genesisPath := utils.GetGenesisPathTest(t, genesisName)
```

and use it like

```go
generateTestValidRemarkTxns(t, utils.GssmrGenesis, ...)
```